pkg/envoy/ads/response_test.go

```diff
@@ -33,11 +33,11 @@ var _ = Describe("Test ADS response functions", func() {
 	fakeCertManager, err := certificate.FakeCertManager()
 	Expect(err).ToNot(HaveOccurred())
 	proxyUUID := uuid.New()
-	proxySvcAccount := tests.BookstoreServiceAccount
+	proxySvcID := tests.BookstoreServiceIdentity
```

Comment:
`proxySvcID` is still functionally a service account right? This is just de-coupling from Kubernetes right?

Reply:
yup, that's right!